src/System.Net.Security/tests/FunctionalTests/SslStreamSystemDefaultsTest.cs

```diff
@@ -95,6 +91,32 @@ public TestBase()
                 }
             }
 
+            private bool ServerCertCallback(object sender, X509Certificate certificate, X509Chain chain, SslPolicyErrors sslPolicyErrors)
+            {
+                switch (sslPolicyErrors)
+                {
+                    case SslPolicyErrors.None:
+                    case SslPolicyErrors.RemoteCertificateChainErrors:
+                    case SslPolicyErrors.RemoteCertificateNameMismatch:
+                        return true;
+                    case SslPolicyErrors.RemoteCertificateNotAvailable:
+                        if (PlatformDetection.IsWindows7 && !Capability.IsTrustedRootCertificateInstalled())
```

Comment:
I agree with @bartonjs on this.

Also,

>This is because of a behavioral difference between win7 and win8+ with regard to how the trusted issuers list is exchanged between the server and the client.

This difference in the OS behavior is regarding the sending of the 'trusted issuers list'.  But that list is all about issuers that the server trusts with respect to the client sending a client certificate.  The 'trusted issuers list' has nothing to do with the client validating the server certificate nor the trusted root of the server certificate chain.